### ISSUE TITLE
Issues upgrading from 3.2.x with latest ORCID profile plugin

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#7749]
**Describe the bug**
When upgrading to 3.3.0-9 from an OJS 3.2.x installation that has the latest (1.1.2.26) version of the ORCID profile plugin installed, the upgrade tool fails with the following message:
```
[code: Installer Installer::addPluginVersions]
<h1>You are trying to downgrade the product &quot;orcidProfile&quot; from version [1.1.2.26] to version [1.1.2.25]. Downgrades are not supported.</h1>ojs2: You are trying to downgrade the product "orcidP
rofile" from version [1.1.2.26] to version [1.1.2.25]. Downgrades are not supported.
```
This appears to happen because the version number for orcidProfile that ships in 3.3.0-9 (1.1.2.25) is lower than the latest 3.2.x version (1.1.2.26).

Only increasing version numbers are supported during upgrade.

**Comment by asmecher** [comment]
New release of ORCID profile has been put out and the submodule in `stable-3_3_0` is updated.

**Comment by dagosalas** [comment]
This problem was already presented in version 3.3.0.8, I reported it in this [post](https://forum.pkp.sfu.ca/t/error-orcid-plugin-when-update-to-3-3-0-8-try-to-downgrade-it/70929/2).

**Comment by asmecher** [comment]
Thanks, @dagosalas, I've cross-linked from that (closed) thread. It should be resolved now with 3.3.0-10.
